coalib/results/AbsolutePosition.py

```diff
@@ -0,0 +1,56 @@
+from coalib.results.TextPosition import TextPosition
+from coalib.misc.Decorators import enforce_signature
+
+
+class AbsolutePosition(TextPosition):
+
+    @enforce_signature
+    def __init__(self,
+                 text: (tuple, list, None)=None,
+                 position: (int, None)=None):
+        """
+        Creates an AbsolutePosition object that represents the index of a
+        character in a string.
+
+        :param text:     The text containing the character.
+        :param position: Position identifying the index of character
+                         in text.
+        """
+        line = column = None
+        if position and text:
+            line, column = calc_line_col(text, position)
+        self._text = text
+        self._position = position
+        super(AbsolutePosition, self).__init__(line, column)
+
+    @property
+    def position(self):
+        return self._position
+
+
+def calc_line_col(text, position):
+    """
+    Creates a tuple containing (line, column) by calculating line number
+    and column in the text, from position. Uses \\n as the newline
+    character. Line and column start from 1.
+
+    :param text:          A tuple/list of lines in which position is to
+                          be calculated.
+    :param position:      Position of character to be found in the
+                          (line, column) form.
```

Comment:
does the position include `\n` chars? I think it does right? So `"\n".join(text)[position:position+1]` should get that char right? Then we want position to start with 0?